COLLABORATOR_GUIDE.md

```diff
@@ -41,36 +53,46 @@ Collaborators or additional evidence that the issue has relevance, the
 issue may be closed. Remember that issues can always be re-opened if
 necessary.
 
-[**See "Who to CC in issues"**](./doc/onboarding-extras.md#who-to-cc-in-issues)
+[See "Who to CC in issues"](./doc/onboarding-extras.md#who-to-cc-in-issues)
 
 ## Accepting Modifications
 
 All modifications to the Node.js code and documentation should be
 performed via GitHub pull requests, including modifications by
-Collaborators and TSC members.
+Collaborators and TSC members. A pull request must be reviewed, and usually
+must also be tested with CI, before being landed into the codebase.
+
+### First Time Contributions
+
+Courtesy should **always** be shown to individuals submitting issues and pull
```

Comment:
I really liked that it was the first sentence in this guide, just after the TOC.

Reply:
Hmm...I think we can just move the section up because this does not seem to be about "Accepting Modifications" anyway.

Reply:
On a second thought, I don't think this paragraph should appear before explaining that "a collaborator can manage issues and pull request", it looks a bit surprising to explain "how you should handle something" before saying "you can handle something"